AlertaDengue/dados/charts/states.py

```diff
@@ -5,26 +5,45 @@
 import plotly.graph_objs as go
 import pandas as pd
 from plotly.subplots import make_subplots
+from django.utils.translation import gettext as _
+
+from copy import deepcopy
 
 
 class ReportStateCharts:
+    """Charts used by Report State."""
+
     @classmethod
-    def create_tweet_chart(cls, df: pd.DataFrame, year_week, disease: str):
+    def create_tweet_chart(
+        cls, df: pd.DataFrame, year_week: int, disease: str
+    ) -> str:
         """
-        :param df:
-        :param year_week:
-        :param disease:
-        :return:
+        Create chart with tweet information.
+        Parameters
+        ----------
+        df : pd.DataFrame
+            Dataframe with tweet information
+        year_week : int
+            Year and week desired filter e.g.: 202002
+        disease : str, {'dengue', 'chik', 'zika'}
+            Disease name
+        Returns
+        -------
+        str
+            HTML with Plotly chart.
         """
+        df = deepcopy(df)
         ks_cases = ['casos notif. {}'.format(disease)]
 
+        # TODO: check this code
```

Comment:
why you have this `TODO` here? if is something you need to resolve now, please do it.
if you added this for the future, add more description please.

Reply:
> why you have this `TODO` here? if is something you need to resolve now, please do it.
> if you added this for the future, add more description please.

This TODO was to check with pdb why tweet charts were in error.
This is OK!

Reply:
Ok, so add more information there pls ;)